daiquiri/core/management/commands/find_users.py

```diff
@@ -0,0 +1,142 @@
+import csv
+import re
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from allauth.account.models import EmailAddress
+from daiquiri.auth.models import Profile
+
+# from rdmo.projects.models import Membership
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("--id", default=".*", help="find users by id")
+        parser.add_argument("--email", default=".*", help="find users by email")
+        parser.add_argument("--username", default=".*", help="find users by username")
+        parser.add_argument(
+            "--first_name", default=".*", help="find users by first name"
+        )
+        parser.add_argument("--last_name", default=".*", help="find users by last name")
+        parser.add_argument(
+            "--email_verified",
+            default=".*",
+            help="filter users by email_verified, requires argument, supports: true/false/1/0",
+        )
+        parser.add_argument(
+            "--profile_pending",
+            default=".*",
+            help="filter users by profile_pending, requires argument, supports: true/false/1/0",
+        )
+        parser.add_argument(
+            "--profile_confirmed",
+            default=".*",
+            help="filter users by profile_confirmed, requires argument, supports: true/false/1/0",
+        )
+        parser.add_argument(
+            "-p",
+            "--print",
+            action="store_true",
+            help="print found users, don't save them to csv",
+        )
+        parser.add_argument(
+            "-o",
+            "--output_file",
+            default="found_users.csv",
+            help="output file, default is 'found_users.csv'",
+        )
+
+    def save_csv(self, data, filename):
+        if len(data) > 0:
+            data_file = open(filename, "w")
+            csv_writer = csv.writer(data_file)
+            csv_writer.writerow(list(data[0].keys()))
```

Comment:
this is writing the columns, right? I think there is a csv_writer method for that. `write_header`, it is cleaner since you can better control the formatting.